Edit.py

```diff
@@ -0,0 +1,38 @@
+import PrintList
+
+def edit(to_Do):
+  # Print List
+  PrintList.print_list(to_Do)
+
+  X = input("[Press X if you want to exit]").upper()
```

Comment:
What does `.upper()` do if the user entered a number instead of a character? Do we need to worry about that?

Reply:
It does not have any affect on the program. Will just continue to the next part of the program